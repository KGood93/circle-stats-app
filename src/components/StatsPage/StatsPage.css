.StatsPage {
    display: flex;
    align-items: center;
    flex-direction: column;
    margin-top: 25px;
}

.graph {
    width: 100%;
    font-family: 'Roboto', sans-serif;
}
