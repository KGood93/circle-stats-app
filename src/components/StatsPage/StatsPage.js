import React, {Component} from 'react'
import Graph from '../Graph/Graph'
import StatsTable from '../StatsTable/StatsTable';
import './StatsPage.css'

class StatsPage extends Component {

    //API Fetch

    render() {
        return (
            <div className="StatsPage">
                <div className="graph">
                    <Graph />
                </div>
                <div className="textStats">
                    <StatsTable />
                </div>
            </div>
        )
    }
}

export default StatsPage