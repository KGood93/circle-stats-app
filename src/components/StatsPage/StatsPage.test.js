import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import StatsPage from './StatsPage';

describe('StatsPage Component', () => {
    it(`renders without crashing`, () => {
        const wrapper = shallow(<StatsPage />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});