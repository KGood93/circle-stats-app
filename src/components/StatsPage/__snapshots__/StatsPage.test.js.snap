// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatsPage Component renders without crashing 1`] = `
<ContextProvider
  value={
    Object {
      "data": Array [],
    }
  }
>
  <div
    className="StatsPage"
  >
    <div
      className="graph"
    >
      <Graph
        workingData={
          Object {
            "data": Array [],
          }
        }
      />
    </div>
    <div
      className="textStats"
    >
      <StatsTable
        workingData={
          Object {
            "data": Array [],
          }
        }
      />
    </div>
  </div>
</ContextProvider>
`;
